### PR TITLE
add trello app to tools page; some edits; lossless compress tools images

### PR DESCRIPTION
**Pull request by ghost**
ready for review! thanks.

### CODE REVIEW

File: content/posts/2017/01/trello-zettelkasten-implementation.md [hunk 1]
```diff
@@ -83,7 +83,7 @@ Add cards via the “Add a card…” input at the bottom of lists. You can also
 * Press `n` to add a new card after the currently selected card. This key shortcut only works if there is a card currently selected. I wish it worked all the time.
 * Press `shift-enter` while submitting a card to open it immediately after creating it.
 
-<%= insert_rel_image file: "add-card.png", title: "Adding a card", caption: %{Click here to add a card or press 'n'} %>
+<%= insert_rel_image file: "add-card.png", title: "Adding a card", caption: %{Click here to add a card. Or press 'n'} %>
```
Comment (ghost) [inline content/posts/2017/01/trello-zettelkasten-implementation.md:86]: this is the last edit. i promise. 😉

**Comment by DivineDominion** [comment]
Cool, thanks! What tool did you use to compress the images?

**Comment by ghost** [comment]
this time: https://kraken.io/web-interface otherwise I use https://github.com/Kilian/Trimage - for mac it's https://imageoptim.com/mac

**Comment by DivineDominion** [comment]
thanks!

Changes went live already 👍

**Pull request merged**